### ISSUE TITLE
hyenaDNA for regression?

### ISSUE THREAD

**Issue by pierrj** [HazyResearch/hyena-dna#47]
I would like to use the pre-trained hyenaDNA model for a regression task. I took your code from Google colab and have adapted it successfully for a classification task on the same dataset. However, when I attempted to modify my code for regression (essentially just setting n_classes = 1 and changing the dataloader to not encode the labels) the model train runs without errors but the training loss does not decrease at all over time.

Have you tried to use hyenaDNA for a regression task before? I can provide more code if you expect it to work for this type of task

**Comment by exnx** [comment]
We have not!  But it should work!  Good luck :)
